src/components/ui/separator.tsx

```diff
@@ -0,0 +1,3 @@
+export default function Separator() {
+  return <div className="my-4 border-t border-gray-300"></div>;
+}
```

Comment:
This implementation conflicts with original shadcn's implementation of Separator.


<img width="1099" alt="image" src="https://github.com/user-attachments/assets/d229834b-fb46-4f4a-bb4a-9bddf68854b2">

Reply:
I tried adding the `Shadcn Separator` , but it caused deployment issues
![Screenshot 2024-10-28 213226](https://github.com/user-attachments/assets/2dffbd18-8f72-41d7-972a-0239c093cadf)

Reply:
When installing shadcn's separator, it'd also install an additional package from radix-ui. I don't see any package.json changes in the commit where you've originally attempted to install shadcn's separator